### ISSUE TITLE
Break button press has no effect OS X El Capitan 10.11.15

### ISSUE THREAD

**Issue by RobertJEE** [Cu3PO42/KeySAVe#116]
I followed the instructions in the documentation and went to break using 16 and 165. I press the break button and the button's visual gradient change when pressed tells me that the program is registering the button press.

Nothing happens after this though. No message, no confirmation, no loading progress of any kind, program doesn't hang or anything either, its like the button isn't tied to a command.

I also tried 16/165 and 16.sav/165.sav since It wasn't 100 percent clear if the file extension needed to be preserved.

**Comment by Cu3PO42** [comment]
That's very odd behavior! Are you using version 1.2.1? Could you please share the log from `~/Library/Application Support/keysave.log`?

**Comment by RobertJEE** [comment]
No problem, here ya go. Please see attached.

Date: Sat, 2 Jul 2016 11:22:07 -0700

**Comment by Cu3PO42** [comment]
Yeah, that's not supposed to happen this way. Could you send me the `16` and `165` via e-mail to cu3po42@gmail.com so I can debug this issue?

**Comment by RobertJEE** [comment]
sent to your email.

Date: Sat, 2 Jul 2016 15:05:47 -0700

**Comment by Cu3PO42** [comment]
It appears the first Pokémon in Box 1 is corrupted. It has a OT language value of 216, valid values are 1 through 8. Could it be a bad hack potentially? If you catch 6 new Pokémon and repeat the process, it should work.

I'll add an error message about this in the future, though.

**Comment by RobertJEE** [comment]
Awesome thanks for the response! No it wasn't a hack or anything since i don't know how to hack/gen anything. When I went to fill the box rows I made sure they all had my trainer name so o I knew they were mine. 

I guess It's possible that I got that Pokemon from wonder trade from someone with the exact same trainer name as me. Possibly that person hacked or genned it. 

Thanks for looking into it! It'd be nice in the future if something is off like this the program would give some kind of warning or error message.
